app/views/foods/index.html.erb

```diff
@@ -0,0 +1,25 @@
+<div class="table-responsive">
+  <div class="container">
+    <%= link_to 'New Food', new_food_path, class: 'btn btn-primary new-food' %>
+    <table class="table table-striped border">
+      <thead>
+        <tr>
+          <th>Name</th>
+          <th>Price</th>
+          <th>Measurement Unit</th>
+          <th>Actions</th>
+        </tr>
+      </thead>
+      <tbody>
+        <% @foods.each do |food| %>
+        <tr>
+          <td><%= food.name %></td>
+          <td><%= food.price %></td>
+          <td><%= food.measurement_unit %></td>
+          <td><%= link_to 'Delete', food_path(food), method: :delete, class: 'btn btn-outline-secondary' %></td>
+        </tr>
+        <% end %>
+      </tbody>
+    </table>
+  </div>
+</div>
```

Comment:
- [ ] Please make sure that this page won't appear before the user login,  I can see the page before the login action. This has to be replaceable by the login and sign-up page.

![user-sign](https://user-images.githubusercontent.com/55635977/148660182-bb0f2414-8fbe-4a9b-a31e-46c3332650cb.png)

Reply:
This feature is implemented now